weave

```diff
@@ -1066,10 +1081,7 @@ http_call_unix() {
 
 call_weave() {
     check_running $CONTAINER_NAME || return 1
-    if ! http_call $HTTP_ADDR "$@" ; then
-        echo "Call to $CONTAINER_NAME failed." >&2
-        return 1
-    fi
+    http_call $HTTP_ADDR "$@"
```

Comment:
This stuff is surprisingly subtle; in particular after `if ! http_call ...`, `$?` is 0, probably because it is the return code from `!`.